README.rst

```diff
@@ -553,6 +558,41 @@ their hardware ID instead of their name.
 
 If you still want to reference devices by their name, set this to ``False``.
 
+Notifications
+-------------
+
```

Comment:
In this section, I would focus on listing the notification types used by openwisp-controller and take advantage to show some screenshots